gauge-tests/properties/project.properties

```diff
@@ -0,0 +1 @@
+webdriver.chrome.driver =D:/WebDriver/chromedriver/chromedriver23.exe
```

Comment:
the same property was defined somewhere before

Reply:
will remove